clang/lib/AST/CommentParser.cpp

```diff
@@ -89,6 +89,31 @@ class TextTokenRetokenizer {
     }
   }
 
+  /// Extract a template type
+  bool lexTemplate(SmallString<32> &WordText) {
+    unsigned IncrementCounter = 0;
```

Comment:
`BracketCount` would be a better name

Reply:
That's a much better name, fixed and thank you.